src/modules/auth/components/auth-view.jsx

```diff
@@ -8,7 +9,7 @@ import AirbitzLogoIcon from 'modules/common/components/airbitz-logo-icon';
 
 import ComponentNav from 'modules/common/components/component-nav';
 
-import { AUTH_SIGNUP, AUTH_LOGIN, AUTH_IMPORT } from 'modules/app/constants/views';
+import { SIGNUP, LOGIN, IMPORT } from 'modules/app/constants/views';
```

Comment:
In this case it’s used for two unique set of user actions.

Creation of an account — `Sign Up`
Authentication — `Login`